roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -66,60 +94,78 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             -
-              name: "KUBERNETES_TRUST_CERT"
-              value: "true"
+              name: "NODE_NAME"
```

Comment:
The point is if we base on DC, the seeding of the index probably only happens once where based on the pod name it will happen everytime the pod is restarted.